### PR TITLE
Remove Windows diskless VM bug workaround

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1871437

CP for 4.5, 4.6

### CODE REVIEW

**Comment by pkliczewski** [comment]
Looks good to me!

File: modules/virt-creating-configmap.adoc [hunk 1]
```diff
@@ -1,16 +1,9 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 [id='virt-creating-configmap_{context}']
```
Comment (aburdenthehand) [inline modules/virt-creating-configmap.adoc:3]: I know it's unrelated to the PR scope but can you please insert a newline between the metadata and the id. It's important for managing module metadata at scale.
And the quote marks in the id should be ":
[id="virt-creating-configmap_{context}"]

For reference:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

File: modules/virt-creating-configmap.adoc [hunk 2]
```diff
@@ -1,16 +1,9 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
```
Comment (aburdenthehand) [inline modules/virt-creating-configmap.adoc:2]: Less important than the subsequent comment but line 2 should be a commented new line. Ie: 
```
// Module included in the following assemblies:
// 
// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
```

File: modules/virt-creating-configmap.adoc (hunk 1 above)
Comment (apinnick) [reply modules/virt-creating-configmap.adoc:3]: OK, but there is no good reason for either of these changes. Not having a line between metadata and ID does not create a problem. Same with quotes.

Also, this module will be removed in CNV 2.5.

File: modules/virt-creating-configmap.adoc (hunk 2 above)
Comment (apinnick) [reply modules/virt-creating-configmap.adoc:2]: Less important? Definitely.

**Comment by apinnick** [comment]
@aburdenthehand  Module updated

**Pull request merged**